src/Makefile

```diff
@@ -54,7 +55,7 @@ run-cppcheck:
 
 install: all
 	install -d $(DESTDIR)$(SBINDIR)
-	install -s dinit dinitctl $(SHUTDOWN) $(DESTDIR)$(SBINDIR)
+	install -s --strip-program=$(STRIP) dinit dinitctl $(SHUTDOWN) $(DESTDIR)$(SBINDIR)
```

Comment:
`--strip-program` is not portable I'm afraid. On BSD systems you'd need to set the `STRIP` environment variable instead.

Reply:
I would suggest adding a variable for install instead (ie INSTALL), in the  which can be set to "install -s" or "install -s --strip=..." etc. It could be set by default to "install -s" at the top of the main Makefile via `INSTALL ?= install -s`.

Reply:
Good idea!

Reply:
> I would suggest adding a variable for install instead (ie INSTALL), in the which can be set to "install -s" or "install -s --strip=..." etc. It could be set by default to "install -s" at the top of the main Makefile via `INSTALL ?= install -s`.

Turns out it won't work:
```
INSTALL ?= install -s

install -s -d ./install/sbin
install: the strip option may not be used when installing a directory
```

Reply:
Hmm. Maybe just set:

`INSTALL ?= install`

then have:

```
install: all
	$(INSTALL) -d $(DESTDIR)$(SBINDIR)
	$(INSTALL) -s dinit dinitctl $(SHUTDOWN) $(DESTDIR)$(SBINDIR)
```

And you can set:

`INSTALL = install --strip-program=...`

... in your config?

Reply:
It will work but install will output a warning saying the option has no effect with -d